src/Nethermind/Nethermind.Evm/CodeAnalysis/IL/VirtualMachineDependencies.cs

```diff
@@ -39,18 +39,18 @@ public static void UpdateGasUp(long refund, ref long gasAvailable)
             gasAvailable += refund;
         }
 
-        private static bool ChargeAccountAccessGasWithDelegation(ref long gasAvailable, ICodeInfoRepository codeInfoRepository, IWorldState state, EvmState vmState, Address address, IReleaseSpec spec, bool chargeForWarm = true)
+        private static bool ChargeAccountAccessGasWithDelegation(ref long gasAvailable, ICodeInfoRepository codeInfoRepository, IWorldState state, EvmState vmState, Address address, IReleaseSpec spec, ref ITxTracer txTracer, bool chargeForWarm = true)
```

Comment:
let's turn `ref ITxTracer txTracer` into `ITxTracer txTracer` so that we don't double unwrap it